src/SingleBookingPage/BookingDetail.js

```diff
@@ -89,14 +89,18 @@ class BookingDetail extends React.Component<Props> {
     const { booking } = this.props;
     const departureTime = this.getDepartureByType(booking);
     const arrivalTime = this.getArrivalByType(booking);
-    const departureTimeDelta = this.decideIfIsFutureAndUrgent(departureTime);
-    const arrivalTimeDelta = this.decideIfIsFutureAndUrgent(arrivalTime);
-    const { isFuture, isUrgent, timeDelta } = departureTimeDelta;
+    const departureInfo = this.decideIfIsFutureAndUrgent(departureTime);
+    const arrivalInfo = this.decideIfIsFutureAndUrgent(arrivalTime);
+    const { isUrgent, timeDelta } = departureInfo;
+
+    const isBetweenDepartureAndArrival =
+      !departureInfo.isFuture && arrivalInfo.isUrgent;
+    const showContactInfo = isUrgent || isBetweenDepartureAndArrival;
```

Comment:
Considering you don't seem to use `isBetweenDepartureAndArrival` anywhere else. 
Could we refactor this logic( L96 to L98) just as? :slightly_smiling_face: 
```javascript
const showContactInfo = departureInfo.isUrgent || arrivalInfo.isUrgent;
```

Reply:
What does it mean `L96 to L98`?

Reply:
Oh, I see. Overkill.